### PR TITLE
 Qemu version revert 6.0 to 4.2

### PR DESCRIPTION
**Pull request by gollarx**
Issue with Qemu 6.0 in TPY branch.So changing back to qemu4.2.

Tracked-On: OAM-102241
Signed-off-by: gollarx <ratnakumarix.golla@intel.com>

### CODE REVIEW

**Comment by gollarx** [comment]
Hello zhuo.peng@intel.com and xiao.he@intel.com, please review this PR.

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
+index 57d55ed..667a37c 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -9,7 +9,7 @@ set -eE
+ 
+ #---------      Global variable     -------------------
+ reboot_required=0
+-QEMU_REL="qemu-6.0.0"
```
Comment (zhuopeng-sg) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch:20]: The qemu 6.0 change is introduced by 0002-Switch-CIV-to-use-QEMU-6.0.0.patch. This patch is for ADL and should not be picked up by tpy branch. I checked in TPY branch, the qemu version is still 4.2

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+-    build -b DEBUG -t GCC5 -a X64 -p OvmfPkg/OvmfPkgX64.dsc -D NETWORK_IP4_ENABLE -D NETWORK_ENABLE  -D SECURE_BOOT_ENABLE -D TPM_ENABLE
++    build -b DEBUG -t GCC5 -a X64 -p OvmfPkg/OvmfPkgX64.dsc -D NETWORK_IP4_ENABLE -D NETWORK_ENABLE  -D SECURE_BOOT_ENABLE -DTPM2_ENABLE=TRUE
+     cp Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd ../../../OVMF.fd
+ 
+     if [ -d $CIV_GOP_DIR ]; then
+@@ -184,28 +183,6 @@ function ubu_enable_host_gvt(){
+     fi
+ }
+ 
+-function ubu_enable_host_sriov(){
```
Comment (zhuopeng-sg) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch:65]: The sriov change is introduced by patch 0001-Support-SRIOV-in-setup-and-start-scripts.patch, which should only be merged into ADL branch,. Why does TPY branch pick up this branch?

**Comment by zhuopeng-sg** [comment]
I have conflict when try to apply this PR on top of latest tpy branch. Please correct me if my TPY manifests is not correct.
repo init -u https://github.com/intel-innersource/os.android.bsp.manifests -b celadon/r/tpy/vertical -m r2

Here are the conflicts:
Project ./device/intel/civ/host/vm-manager
Conflicts               0003-SDE-FRP-vm-manager-start_civ.sh-changes.patch
Conflicts               0004-vm-manager-Add-BLKDISCARD-BLKDISCARDZEROES-ioctl-sup.patch
Conflicts               0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch


Project ./vendor/intel/utils
Conflicts               0001-Revert-Upgrade-the-qemu-4.2-to-6.0-patches.patch

Project ./device/intel/civ/host/vm-manager
Conflicts               0012-Support-keybox-provisioning-over-vsock.patch
Conflicts               0013-Update-start_flash_usb.sh-for-keybox-provisioning.patch

I also noticed that the changes you revert are for ADL. They should not be in TPY branch by default.

**Comment by gollarx** [comment]
please check with this manifest PR
celadon/r/tpy/vertical --> https://github.com/intel-innersource/os.android.bsp.manifests/pull/490

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0003-SDE-FRP-vm-manager-start_civ.sh-changes.patch [hunk 3]
```diff
@@ -13,7 +13,7 @@ Signed-off-by: Xiao He <xiao.he@intel.com>
  2 files changed, 125 insertions(+), 45 deletions(-)
 
 diff --git a/scripts/start_civ.sh b/scripts/start_civ.sh
```
Comment (zhuopeng-sg) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0003-SDE-FRP-vm-manager-start_civ.sh-changes.patch:15]: Base on "https://github.com/projectceladon/vm_manager/pull/171", do we also need to revert "-cpu host,-waitpkg \" in start_civ.sh file?

**Comment by zhuopeng-sg** [comment]
In the commit message, please include the revert PR link. 
https://github.com/projectceladon/vendor-intel-utils/pull/1406
https://github.com/projectceladon/vm_manager/pull/171

Also please update JIRA to give more details. Attach the email you forwarded to me.

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0003-SDE-FRP-vm-manager-start_civ.sh-changes.patch (hunk 3 above)
Comment (gollarx) [reply bsp_diff/caas/device/intel/civ/host/vm-manager/0003-SDE-FRP-vm-manager-start_civ.sh-changes.patch:15]: yes, we have to revert this as well

**Comment by gollarx** [comment]
Updated the Commit message.

**Review by zhuopeng-sg — APPROVED** [review]
Seems OK to me.

**Review by XiaoHeSG — APPROVED** [review]

**Review by XiaoHeSG — APPROVED** [review]

**Review by zhuopeng-sg — APPROVED** [review]

**Pull request merged**